### ISSUE TITLE
Ignored files display 'untracked' icons

### ISSUE THREAD

**Issue by melriffe** [jisaacks/GitGutter#353]
The 'untracked' icon is the green plus sign, yes?

I'm not sure what changed, but my ignored files are showing up as 'untracked'. What information can I supply to help define and debug this issue?

For starters:
Mac OS X 10.9.5
git version 2.11.0 (managed via homebrew)
Sublime Text 3, Stable Chanel, Build 3126
Git Gutter 1.4 (I reinstalled to make sure I had the latest)

Thanks!

**Comment by deathaxe** [comment]
The intended icons are:

```
* = untracked (gray)
x = ignored (gray)
+ = new tracked file (green)
```
Will be fixed in next release by PR #338

**Comment by melriffe** [comment]
@deathaxe Thanks for the quick feedback, and correction. :-/

I'll report back after next release. 👍
